parser/match_operations_test.go

```diff
@@ -105,6 +105,152 @@ func TestMatchOperations(t *testing.T) {
 			},
 			err: false,
 		},
+		"simple transfer (with OppositeOrZeroAmounts) and opposite value amounts": {
```

Comment:
For both added tests, the code returns the expected Matches output, but in reverse order (i.e. the below passes for the first added test, similarly reversed output passes for the second). Are the matches with this changed order still a correct output?

Possibly relevant: the code passes the first test and produces non-reversed Matches when using NegativeAmountSign and PositiveAmountSign instead of AnyAmountSign, so the sign may impact this ordering.

```
			matches: []*Match{
				{
					Operations: []*types.Operation{
						{
							Account: &types.AccountIdentifier{
								Address: "addr2",
							},
							Amount: &types.Amount{
								Value: "100",
							},
						},
					},
					Amounts: []*big.Int{big.NewInt(100)},
				},
				{
					Operations: []*types.Operation{
						{
							Account: &types.AccountIdentifier{
								Address: "addr1",
							},
							Amount: &types.Amount{
								Value: "-100",
							},
						},
					},
					Amounts: []*big.Int{big.NewInt(-100)},
				},
			},
			err: false,
		},
```